src/App.js

```diff
@@ -1,14 +1,20 @@
 import "./App.css";
+import { Routes, Route } from "react-router-dom";
 import {Navbar,Footer} from "./components/components";
-// import Footer from "./components/Footer";
+import Login from "./pages/Login";
 import Products from "./pages/Products";
+import Signup from "./pages/Signup";
 
 function App() {
   return (
     <div className="App">
       <Navbar/>
       <main>
-        <Products/>
+        <Routes>
+          <Route path="/" element={<Login />} />
+          <Route path="signup" element={<Signup/> } />
+          <Route path="products" element={<Products/>}/>
+        </Routes>
```

Comment:
is it purposely you have put Products component out of route and also mapped in route.if it is part of your feature you can ignore it

Reply:
No, I have moved it in route..initially, it was out.